tests/pass-dep/libc/libc_syscall_eventfd2.rs

```diff
@@ -0,0 +1,11 @@
+//@only-target: linux
+
+// This is a test for calling eventfd2 through a syscall.
+fn main() {
+    let initval = 0 as libc::c_uint;
+    let flags = (libc::EFD_CLOEXEC | libc::EFD_NONBLOCK) as libc::c_int;
+
+    let result = unsafe { libc::syscall(libc::SYS_eventfd2, initval, flags) };
+
+    assert_eq!(result, 3); // The first FD provided would be 3.
```

Comment:
Just a nit, will it be better if the assert here is``assert_ne!(result, -1)``? Although the current implementation will always give 3 as first fd value, but imo this can change any time. We just need to make sure that it won't be an error, so any value other than -1 should pass the assert.

Reply:
Okay.